controller.js

```diff
@@ -1,30 +1,41 @@
 
-const returnTopics = require("./model")
+const {returnTopics,returnArticle} = require("./model")
 const fs = require('fs')
-const data = require('./endpoints.json')
+const endpointsData = require('./endpoints.json')
 
 
 
 function getTopics(req, res, next){
-    
-returnTopics()
-.then(data => {
-    
+    returnTopics()
+    .then(data => {
     res.status(200).send({'topics': data})
-})
-.catch(next)
+    })
+    .catch(next)
+    }
+
+function getArticle(req, res, next){
+    const articleId =req.params.article_id
+    console.log(articleId)
+    if(isNaN(articleId))
+    {
+    res.status(400).send({msg: "Bad Request"})
+    }
+    returnArticle(articleId)
+    .then((articleData)=>{
+    if(articleData.length === 0){
```

Comment:
The convention would be to handle this check in the model - using a promise.reject to throw an error if the length of the response from the sql query was empty.